integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/KafkaPublishingSteps.java

```diff
@@ -129,7 +129,9 @@ private static ScadaMeasurementPublishedEvent createExpectedMessage(
     names.add(new Name(new NameType("gisbehuizingnummer"), substationIdentification));
     names.add(new Name(new NameType("versie"), version));
     names.add(new Name(new NameType("msr naam"), substationName));
-    names.add(new Name(new NameType("bay positie"), bayPosition));
+    if (fieldCode != null) {
```

Comment:
If fieldCode is not nullable, it is probably not needed to do a null check here, although it won't harm either...